gnomad/resources/grch37/reference_data.py

```diff
@@ -287,3 +360,36 @@ def get_truth_ht() -> hl.Table:
         .repartition(200, shuffle=False)
         .persist()
     )
+
+
+gtex_rsem = VersionedTableResource(
+    default_version="v7",
+    versions={
+        "v7": GnomadPublicTableResource(
+            path="gs://gnomad-public-requester-pays/resources/grch37/gtex_rsem/gtex_rsem_v7.mt",
+            import_func=_import_gtex_rsem,
+            import_args={
+                "gtex_path": "gs://gcp-public-data--gnomad/resources/grch37/gtex/bulk-gex_v7_rna-seq_GTEx_Analysis_2016-01-15_v7_RSEMv1.2.22_transcript_tpm.txt.gz",
+                "meta_path": "gs://gcp-public-data--gnomad/resources/grch37/gtex/annotations_v7_GTEx_v7_Annotations_SampleAttributesDS.txt.gz",
+                "min_partitions": 1000,
+                "reference_genome": "GRCh37",
+            },
+        ),
+    },
+)
+
+gencode_cds = VersionedTableResource(
+    default_version="v19",
+    versions={
+        "v19": GnomadPublicTableResource(
+            path="gs://gnomad-public-requester-pays/resources/grch37/gencode_cds/gencode_v19_cds.ht",
+            import_func=_import_gencode_cds,
+            import_args={
+                "gtf_path": "gs://gcp-public-data--gnomad/resources/grch37/gencode/gencode.v19.annotation.gtf.gz",
+                "reference_genome": "GRCh37",
+                "force_bgz": True,
+                "min_partitions": 12,
```

Comment:
why 12?

Reply:
I saw in Conrad's code, he used 12, based on the size of the file, it seems reasonable.

Reply:
ok, sounds fine, was just surprised it wasn't just put as 10 or 15, 12 just seems like an odd choice

Reply:
I can change it to 10, 12 might be a choice of laptop cores.